settings.d/ansible.yml.example

```diff
@@ -1,4 +1,9 @@
 ---
 :enabled: true
 :working_dir: '~/.foreman-ansible'
+:vcs_integration: true
+:static_roles_paths: [
+  '/etc/ansible/roles',
+  '/usr/share/ansible/roles',
+]
```

Comment:
Does this work for you?
I have a different location where I store my Ansible roles, and setting it here doesn't work for me.

Reply:
Are your paths relative perhaps? I just tested it and it works fine with absolute paths, but not relative ones.

Reply:
My path is `/home/nalfassi/playbooks`. It's also configured in my `/etc/ansible/ansible.cfg` as follows:
`roles_path = /home/nalfassi/playbooks`.

Reply:
And I'm using my localhost as my smart-proxy.